### DISCUSSION TITLE
编程难在哪里

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#21] [5.文章资料]
[编程难在哪里？Hugo 的知乎回答](https://www.zhihu.com/question/528871933/answer/2455672960) ([链接备份](https://archive.is/0dMHw))

难在变化是否可控。复杂度是否可控。

**Comment by kalxd** [comment]
难在没有接口可以调。

**Comment by smallyunet** [comment]
抬个杠，什么叫 “难”

**Reply by adoyle-h** [reply]
给”难“下个简单的定义：
“难”通常用来形容某件事情需要花费更多的精力、时间和努力才能掌握或完成。具体来说，“难”可能涉及到以下方面：

- 知识的复杂性和深度
- 技能的复杂性和难度
- 需要具备的逻辑思考和解决问题的能力
- 时间、精力和资源的投入
- 与个人兴趣、动机和职业目标的匹配程度

在不同的领域和任务中，“难”可能具有不同的含义和标准，需要根据具体情况来判断。
